crates/libtest2/tests/testsuite/mixed_bag.rs

```diff
@@ -50,10 +50,22 @@ fn fly(context: &libtest2::TestContext) -> libtest2::RunResult {
 }
 
 #[libtest2::test]
+#[ignore = "fails"]
 fn bear(context: &libtest2::TestContext) -> libtest2::RunResult {
-    context.ignore_for("fails")?;
     Err(libtest2::RunError::fail("no honey"))
 }
+
+#[libtest2::test]
+#[ignore]
+fn sheep(context: &libtest2::TestContext) -> libtest2::RunResult {
+    Err(libtest2::RunError::fail("got lost blindly following the flock"))
+}
+
+#[libtest2::test]
+#[ignore = "slow"]
+fn horse(context: &libtest2::TestContext) -> libtest2::RunResult {
+    Ok(())
+}
```

Comment:
The tests are kept in sync with libtest2-mimic.

Could you have a commit before this one add these different cases in both sets of tests?  This commit would just be about translated from the API to the attribute